seal.go

```diff
@@ -44,29 +45,48 @@ func makeSealedKeyTemplate() *tpm2.Public {
 		Params:  &tpm2.PublicParamsU{KeyedHashDetail: &tpm2.KeyedHashParams{Scheme: tpm2.KeyedHashScheme{Scheme: tpm2.KeyedHashSchemeNull}}}}
 }
 
+func makeImportableSealedKeyTemplate() *tpm2.Public {
+	tmpl := makeSealedKeyTemplate()
+	tmpl.Attrs &^= tpm2.AttrFixedTPM | tpm2.AttrFixedParent
+	return tmpl
+}
+
 func computeSealedKeyDynamicAuthPolicy(tpm *tpm2.TPMContext, version uint32, alg, signAlg tpm2.HashAlgorithmId, authKey crypto.PrivateKey,
```

Comment:
this probably needs a comment about the different behavior depending on tpm != nil or == nil

Reply:
I've added a comment to this now.